src/components/Footer.vue

```diff
@@ -1,29 +0,0 @@
-<template>
-  <div class="footer">
-    <span>
-      ©
```

Comment:
Damn the copyright is legit

Reply:
LOL, added that just for fun, wasn't sure if you'd notice haha.